core/shared/platform/common/posix/posix_time.c

```diff
@@ -4,6 +4,7 @@
  */
 
 #include "platform_api_vmcore.h"
+#include <winternl.h>
```

Comment:
I wouldn't expect windows-specific includes in the posix implementation - do you need this include for anything?